src/service/post.service.ts

```diff
@@ -35,6 +36,23 @@ class PostService {
   public async createPost(post: Partial<Post>) {
     return this.postRepository.save(post);
   }
+
+  public async isExistedPostLike({ postId, email }: { postId: string; email: string }) {
```

Comment:
해당 타이핑은 아래와 같이 하면 어떨까?

```suggestion
  public async isExistedPostLike({ post, user }: { post: Pick<Post,  'id', user:  Pick<User, 'id'>> }) {
```

위에 처럼 한다면 resolver는 아래 isExistedPostLik가 비즈니스 로직을 구현하기 위해서 필요한 필드값을 알 필요가 없어서 의존성이 낮아지는거 같아.

만약, 나중에 isExistedPostLike 함수가 postId와 email이 아니라 postTitle, userId 인자로 구현한다고 했을때 resolver도 소스 수정을 해야하기 때문에 나중에 수정할 코드가 더 많아지는거지!

추가적으로 위에 처럼 구현한다고 했을때 email보다는 userEmail을 붙이는게 좋을 듯!